mocha_test/eachOf.js

```diff
@@ -274,6 +274,18 @@ describe("eachOf", function() {
         setTimeout(done, 25);
     });
 
+    it('forEachOfLimit 1024 * 1024 synchronous call', function(done) {
```

Comment:
This is minor, but we already have a `forEachOfLimit synchronous` test. Could you rename this to something like `forEachOfLimit no stackoverflow` or `forEachOfLimit no call stack size exceed error` just to differentiate between the two?

Reply:
Could you also add a similar test for `forEachOf` and `forEachOfSeries` to make sure the behaviour is consistent?